pkg/skaffold/survey/config.go

```diff
@@ -39,7 +41,29 @@ var (
 		URL: hatsURL,
 	}
 	// surveys contains all the skaffold survey information
-	surveys = []config{hats}
+	surveys = []config{
+		hats,
+		{
+			id:       "helm",
+			startsAt: time.Date(2021, time.July, 15, 0, 0, 0, 0, time.UTC),
+			expiresAt: time.Date(2021, time.August,
+				14, 00, 00, 00, 0, time.UTC),
+			promptText: "Help improve Skaffold's Helm support by taking our 2-minute anonymous survey!",
```

Comment:
Need to drop the exclamation mark as we add a colon:
```
Help improve Skaffold's Helm support by taking our 2-minute anonymous survey!: run 'skaffold survey --id helm'
```

```suggestion
			promptText: "Help improve Skaffold's Helm support by taking our 2-minute anonymous survey",
```